lib/build.gradle

```diff
@@ -83,3 +89,11 @@ publishing {
         }
     }
 }
+
+task runMemDBServe(type: JavaExec) {
```

Comment:
I added this so I can debug the MemDB plugin via VSCode without having to manually configure class paths